src/_http/index.js

```diff
@@ -131,6 +142,23 @@ HttpClient.prototype.execute = function(options) {
   })
 }
 
+function isValidTraceparentHeader(traceparentHeader) {
+  // Shamelessly copied from shorturl.at/osvwz
+  return /^[\da-f]{2}-[\da-f]{32}-[\da-f]{16}-[\da-f]{2}$/.test(
+    traceparentHeader
+  )
+}
+
+function parseTags(tags) {
+  if (tags === undefined || tags == null || tags == '') return null
+  if (typeof tags != 'object') {
+    throw new Error('Tags must be provided as an object!')
+  }
+  return Object.entries(tags)
+    .map(e => e.join('='))
+    .join(',')
```

Comment:
trim the keys and values?

reject empty/blank keys?

Reply:
Yeah, gonna add validation in a follow-up (including max key/value length, etc.).